dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -1422,9 +1422,31 @@ public void testStarringOrganization() {
         assertEquals(1, organizationsApi.getStarredUsersForApprovedOrganization(organization.getId()).size());
         assertEquals(USER_2_USERNAME, organizationsApi.getStarredUsersForApprovedOrganization(organization.getId()).get(0).getUsername());
 
+        throwsError = false;
+        try {
+            organizationsApi.starOrganization(organization.getId(), body);
```

Comment:
isn't it easier to just throw a fail() under this and get rid of throwsError entirely?